### PR TITLE
Added One TC for the custom ops

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Added one error condition check for the allow_custom_ops set to False

### CODE REVIEW

**Comment by amitsrivastava78** [comment]
@gargn can you pls review the same

**Comment by amitsrivastava78** [comment]
@gargn  can you pls review the PR and let me know your comments

File: tensorflow/lite/python/lite_test.py [hunk 1]
```diff
@@ -751,6 +751,20 @@ def testTFLiteGraphDef(self):
                      output_details[3]['name'])
     self.assertTrue(([1] == output_details[3]['shape']).all())
 
+  def testLiteUnSupportedOps(self):
+    # Tests the object detection model that cannot be loaded in TensorFlow.
+    self._initObjectDetectionArgs()
+
+    # Make the custom_ops as false
```
Comment (gargn) [inline tensorflow/lite/python/lite_test.py]: nit: Change comment to:
"`custom_ops` is not set to `True`"

**Review by gargn — CHANGES_REQUESTED** [review]
All of these changes are to make this test conform to the pattern established in the test cases `testTFLiteGraphDefMissingShape` and `testTFLiteGraphDefInvalidShape`.

File: tensorflow/lite/python/lite_test.py [hunk 2]
```diff
@@ -751,6 +751,20 @@ def testTFLiteGraphDef(self):
                      output_details[3]['name'])
     self.assertTrue(([1] == output_details[3]['shape']).all())
 
+  def testLiteUnSupportedOps(self):
```
[the remark this answers is not in the document]
Comment (amitsrivastava78) [reply tensorflow/lite/python/lite_test.py]: Thanks, i have updated the code as per the comments

File: tensorflow/lite/python/lite_test.py [hunk 3]
```diff
@@ -751,6 +751,20 @@ def testTFLiteGraphDef(self):
...
+    # Tests the object detection model that cannot be loaded in TensorFlow.
```
[the remark this answers is not in the document]
Comment (amitsrivastava78) [reply tensorflow/lite/python/lite_test.py]: Thanks, i have updated the code as per comments

File: tensorflow/lite/python/lite_test.py [hunk 4]
```diff
@@ -751,6 +751,20 @@ def testTFLiteGraphDef(self):
...
+    self._initObjectDetectionArgs()
+
+    # Make the custom_ops as false
+    converter = lite.TFLiteConverter.from_frozen_graph(
+        self._graph_def_file, self._input_arrays, self._output_arrays,
+        self._input_shapes)
+    converter.allow_custom_ops = False
```
[the remark this answers is not in the document]
Comment (amitsrivastava78) [reply tensorflow/lite/python/lite_test.py]: Thanks, i have updated the code as per comments

File: tensorflow/lite/python/lite_test.py (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/python/lite_test.py]: Thanks, i have updated the code as per comments

**Comment by amitsrivastava78** [comment]
@gargn , thanks alot for your comments, i have updated the code as per your comments. Also the TC file is very well written and i just found a missing condition which was not captured in the TC, so added the same. Kindly approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@gargn & @rthadur , I have updated the code as per your comments, kindly check, if everything is ok, kindly approve the PR.

**Comment by amitsrivastava78** [comment]
@rthadur , can you pls update the label to review requested.

**Review by gargn — APPROVED** [review]

**Comment by amitsrivastava78** [comment]
@gargn , thanks for approving the PR, can you please help to get this merged.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@rthadur , can you please help to get this merged.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@gargn thanks for approving the PR.
@rthadur , can you please help to get this merged.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@rthadur , can you please help to get this merged.

Regards
Amit

File: tensorflow/lite/python/lite_test.py [hunk 5]
```diff
@@ -751,6 +751,19 @@ def testTFLiteGraphDef(self):
...
+    self._initObjectDetectionArgs()
+
+    # `custom_ops` is not set to `True`
+    converter = lite.TFLiteConverter.from_frozen_graph(
+        self._graph_def_file, self._input_arrays, self._output_arrays,
+        self._input_shapes)
+
+    with self.assertRaises(lite.ConverterError) as error:
+      converter.convert()
+    self.assertIn(("TOCO failed. See console for info."), str(error.exception))
```
Comment (rthadur) [inline tensorflow/lite/python/lite_test.py]: "TOCO failed" needs to be removed.

**Review by rthadur — CHANGES_REQUESTED** [review]

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**